clicache/src/impl/PdxHelper.hpp

```diff
@@ -46,15 +46,15 @@ namespace Apache
 
           static IPdxSerializable^ DeserializePdx(DataInput^ dataOutput, bool forceDeserialize, const native::SerializationRegistry* serializationRegistry);
 
-          static IPdxSerializable^ PdxHelper::DeserializePdx(DataInput^ dataInput, bool forceDeserialize, int typeId, int length, const native::SerializationRegistry* serializationRegistry);
+          static IPdxSerializable^ DeserializePdx(DataInput^ dataInput, bool forceDeserialize, int typeId, int length, const native::SerializationRegistry* serializationRegistry);
 
           literal Byte PdxHeader = 8;
 
           static Int32 ReadInt32(System::Byte* offsetPosition);
 
           static Int32 ReadInt16(System::Byte* offsetPosition);
 
-					static Int32 PdxHelper::ReadUInt16(System::Byte* offsetPosition);
+					static Int32 ReadUInt16(System::Byte* offsetPosition);
```

Comment:
Still weird indentation if you decide it's worth it

Reply:
please fix this - its a pain that just continues if we don't keep it tidy...

Reply:
Meh - tabs.  Fixed.